README.md

```diff
@@ -226,3 +237,104 @@ The only thing that should seem unfamiliar here is `__resourcePath` -- this is a
 And you can do the same thing in your layout file in order to allow easy links to other docs pages if needed.
 
 > _Nerd Note:_ If you're a real sharp thinker, you may have noticed that it is seemingly impossible for this previous statement to be true, since importing a mdx file requires rendering into its layout, but its layout also requires all the other mdx files, which each require rendering into their layouts, etc. This is what some may call an infinite loop, and it is impossible. In reality, this plugin does a little dirty work under the hood to make this behavior possible. It also injects a babel plugin which extracts the front matter out to separate temporary files, then transforms any front matter imports into importing from the separate file, which breaks the loop.
+
+## Scanning MDX Content
+
+### How It Works
+
+#### Sample `next.config.js` entry:
+
+```js
+// in next.config.js
+withMdxEnhanced({
+  scan: [
+    {
+      pattern: /<SomeComponent.*name=['"](.*)['"].*\/>/,
+      passToLayout: {
+        argumentName: 'someImportantKey',
+        value: matchData => matchData[1] // find the specific entry or simply pass a static value
+      }
```

Comment:
I think the API is not accurate here -- the API exposed in the code and tests is different

Reply:
Great catch, this was outdated.